variables.tf

```diff
@@ -27,5 +27,5 @@ variable "domain_alias" {
 }
 
 variable "mdw_fqdn" {
-  default = "mdw.aepps.com"
+  default = "aeternal.io"
```

Comment:
this does redirect to mainnet.aeternal.io

Reply:
de5241b